lib/storage/base-storage.ts

```diff
@@ -0,0 +1,69 @@
+import {BadRequestError, InternalServerError} from 'errors/apps-sdk-error';
+import {RequestOptions} from 'types/fetch';
+import {isDefined} from 'types/guards';
+import {Options, Token} from 'types/storage';
+import {fetchWrapper} from 'utils/fetch-wrapper';
+import {Logger} from 'utils/logger';
+
+export abstract class BaseStorage {
+    protected logger: Logger;
+
+    protected constructor(protected token?: Token) {
+        this.logger = new Logger('Storage', { mondayInternal: true });
```

Comment:
Where is the usage of the token argument?

Reply:
It's used as a class memeber for classes that extend this abstract base class, like `Storage` with direct usages of `this.token`